stories/Wizard.stories.tsx

```diff
@@ -0,0 +1,174 @@
+import React from "react";
+import range from "lodash/range";
+import uniqueId from "lodash/uniqueId";
+
+import { actions } from "@storybook/addon-actions";
+import { object } from "@storybook/addon-knobs";
+import fakerEN from "faker/locale/en_US";
+import fakerFA from "faker/locale/fa";
+
+import { Button } from "@fluentui/react-northstar";
+
+import { Wizard, WizardDialog } from "../src";
+import { TInputWidth } from "../src/components/Form/FormContent";
+
+const eventsFromNames = actions("onInteraction");
+
+const fake = (template: string) => {
+  return { "en-US": fakerEN.fake(template), fa: fakerFA.fake(template) };
+};
+
+const designLink = {
+  design: {
+    type: "figma",
+    url:
+      "https://www.figma.com/file/EOsbapNvZgEwcA1mShswfh/Microsoft-Teams-UI-Kit-Community?node-id=388%3A8",
+  },
+};
+
+export default {
+  title: "UI Templates/Wizard",
+  component: Wizard,
+};
+
+const wizardKnobGroupID = "Wizard";
+const kitchenSinkConfig = {
+  stepTitles: range(8).map((_) => fake("{{commerce.product}}")),
+  trigger: <Button content="↗️" />,
+  activeStepIndex: 2,
+  activeStep: {
+    submit: {
+      en: "Next",
+      fa: "بعد",
+    },
+    back: {
+      en: "Back",
+      fa: "قبلی",
+    },
+    cancel: {
+      en: "Cancel",
+      fa: "لغو",
+    },
+    headerSection: {
+      title: fake("{{company.catchPhrase}}"),
+      preface: fake("{{lorem.sentences}}"),
+    },
+    sections: [
+      {
+        title: fake("{{company.catchPhrase}}"),
+        preface: fake("{{lorem.sentence}}"),
+        inputGroups: [
+          {
+            type: "text-inputs" as "text-inputs",
```

Comment:
"text-inputs" as "text-inputs", "text" as "text" and etc.
Could you create a Type or Enum for those valuse. Code will be much cleaner and IDE could show all available options for current property.

Reply:
I will investigate… I've noticed in the TS ecosystem, this is not an unusual pattern, but I agree the enum pattern is better where we can offer it.